lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -633,3 +646,62 @@ export class ResponseHandler {
         };
     }
 }
+
+export function buildAccountToCache(
```

Comment:
This logic is duplicated in `ResposneHandler`, `NativeInteraction` and `TokenCache.loadAccount` so I factored it out into a function that is exported here.